# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/unifdef/hotftix-reinstall.patch
# Copyright (C) 2023 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- unifdef-2.12/Makefile.vanilla	2023-05-14 20:31:05.286994407 +0200
+++ unifdef-2.12/Makefile	2023-05-14 20:31:34.009996481 +0200
@@ -28,7 +28,7 @@
 	: manual
 	install -m 755 -d  ${man1dest}
 	install -m 644 unifdef.1  ${man1dest}/
-	ln -s unifdef.1  ${man1dest}/unifdefall.1
+	ln -sf unifdef.1  ${man1dest}/unifdefall.1
 
 clean:
 	rm -f unifdef version.h
